### PR TITLE
CNV#34780: wasp for swap/memory overcommitment RN

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.17 only (RN)

Issue: [CNV-34780](https://issues.redhat.com//browse/CNV-34780)

Link to docs preview: https://83289--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-17-release-notes.html#virt-4-17-virtualization

QE review:
- [x] QE has approved this change.

Additional information: ~Since we don't currently have this documented downstream, I'm wondering if we should link to https://github.com/openshift-virtualization/wasp-agent/blob/main/docs/configuration.md (would need support and PM approval to include a github link). @stu-gott @djfjeff - thoughts?~ (talked to Stu)

### CODE REVIEW

**Comment by stu-gott** [comment]
LGTM

File: virt/release_notes/virt-4-17-release-notes.adoc [hunk 1]
```diff
@@ -68,6 +68,14 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-17-virtualization"]
 === Virtualization
 
+//CNV-34780 GA: Safe memory overcommitment using WASP
+* You can now increase VM workload density on nodes by overcommitting memory (RAM) with the `wasp-agent`. The wasp agent assigns swap resources to worker nodes and manages pod evictions when nodes are at risk.
```
Comment (adellape) [inline virt/release_notes/virt-4-17-release-notes.adoc:72]: Looks like the first instance of "virtual machine (VM)" is in the bullet after this one (line 80). So maybe here:
```suggestion
* You can now increase virtual machine (VM) workload density on nodes by overcommitting memory (RAM) with the `wasp-agent`. The wasp agent assigns swap resources to worker nodes and manages pod evictions when nodes are at risk.
```
And change the instance on line 80 to just "VM"?

Comment (ousleyp) [reply virt/release_notes/virt-4-17-release-notes.adoc:72]: @adellape Ah, actually we have "virtual machines (VMs)" on line 17, and it's written as VM on line 66. I think we struggle with how often to write it out. I might leave it as-is in this PR.

**Pull request merged**